mumbleBot.py

```diff
@@ -464,20 +464,16 @@ def download_music(self, index=-1):
                     self.log.info("bot: download attempts %d / %d" % (i+1, attempts))
                     try:
                         ydl.extract_info(url)
-                        if os.path.exists(mp3):
-                            if os.stat(mp3).st_size > 0:
-                                music['ready'] = "yes"
-                                download_succeed = True
-                                break
-                            else:
-                                os.remove(mp3)
-                        else:
-                            self.log.error("bot: download failed: file not existed or file size is 0.")
```

Comment:
Can you keep the log ? it's not present on you commit. It's easier to debug in case of problem.

Reply:
https://github.com/azlux/botamusique/pull/86/files#diff-7fc1f18100d4cbfd733b4ddd2b29e805L478 logs the download failure already?